content/applications/general/email_communication/email_servers.rst

```diff
@@ -187,6 +187,44 @@ By default, inbound messages are fetched every 5 minutes in Odoo on-premise.
    :menuselection:`Settings --> Technical --> Automation --> Scheduled Actions` and look for
    :guilabel:`Mail: Fetchmail Service`.
 
+Set up different outgoing email servers for a multi-company environment
+=======================================================================
+
+The :guilabel:`FROM Filtering` field allows for the use of a specific outgoing email server
+depending on the :guilabel:`From` email address that Odoo is sending on behalf of. Access this
```

Comment:
After the first sentence (`The...on behalf of`), I would add a real-life example (1-3 sentences) of why a company might want to use this feature. The section header says it's for multi-company environments. Try to simplify for the reader and help them visualize how they would apply this section's knowledge to their business.

Reply:
@mivu-odoo I've added two separate sentences: 1. to improve deliverability  2. In a multi-company environment

The title seemed a bit too specific and I've made it more general as multi-company sending is not the only application. I've changed it to: `Utilizing the FROM Filter on an outgoing email server`